backend/workflow_manager/endpoint/exceptions.py

```diff
@@ -74,8 +72,8 @@ class BigQueryTableNotFound(APIException):
 
 
 class UnstractDBException(APIException):
-    status_code = 400
     default_detail = "Error creating/inserting to database. "
 
-    def __init__(self, code: Any = status_code, detail: str = default_detail) -> None:
-        super().__init__(detail=detail, code=code)
+    def __init__(self, detail: str = default_detail) -> None:
+        status_code = 400
```

Comment:
@kirtimanmishrazipstack can you justify again on why this is a 400 error? Is it really a `BadRequest`?

Reply:
@kirtimanmishrazipstack  this should be 5XX error.